src/Rules/Library/LocalizedLandmarkTypeExcludesPrivateUnicodeCharacters.cs

```diff
@@ -8,21 +8,21 @@
 
 namespace Axe.Windows.Rules.Library
 {
-    [RuleInfo(ID = RuleId.LocalizedLandmarkTypeExcludesSpecialCharacters)]
-    class LocalizedLandmarkTypeExcludesSpecialCharacters : Rule
+    [RuleInfo(ID = RuleId.LocalizedLandmarkTypeExcludesPrivateUnicodeCharacters)]
+    class LocalizedLandmarkTypeExcludesPrivateUnicodeCharacters : Rule
     {
-        public LocalizedLandmarkTypeExcludesSpecialCharacters()
+        public LocalizedLandmarkTypeExcludesPrivateUnicodeCharacters()
         {
-            this.Info.Description = Descriptions.LocalizedLandmarkTypeExcludesSpecialCharacters;
-            this.Info.HowToFix = HowToFix.LocalizedLandmarkTypeExcludesSpecialCharacters;
+            this.Info.Description = Descriptions.LocalizedLandmarkTypeExcludesPrivateUnicodeCharacters;
```

Comment:
Is there a particular reason we aren't reusing the `Descriptions.PropertyExcludesPrivateUnicodeCharacters` and `HowToFix.PropertyExcludesPrivateUnicodeCharacters` strings that the other ExcludesPrivateUnicodeCharacters rules use ([example](https://github.com/microsoft/axe-windows/blob/master/src/Rules/Library/NameExcludesPrivateUnicodeCharacters.cs))?

Reply:
The SpecialCharacters rule has a fairly generic string for Description but a very specific string for HowToFix. I stayed with the existing pattern but am more than willing to change the formatting. @RobGallo, any preferences?

Reply:
If the text is generic enough then using a single string setting would be okay . usually the text contains the name of the property in question; and I think that may be useful.

Reply:
That makes sense. It seems like in this case, the Descriptions string (`Descriptions.PropertyExcludesPrivateUnicodeCharacters`) should be reused because it is identical to the new string, but the HowToFix string needs to be unique because it contains extra info. Does that sound right?